src/examples/candid-health/send-to-candid.ts

```diff
@@ -0,0 +1,350 @@
+    external_id: getReferenceString(encounter), // Use the `Encounter/<id>` as the external id
+    date_of_service: extractDate(encounter.period?.start),
+    end_date_of_service: extractDate(encounter.period?.end),
+    patient_authorized_release: true,
+    benefits_assigned_to_provider: true,
+    provider_accepts_assignment: true,
+    appointment_type: encounter.type?.[0]?.text || '',
+    do_not_bill: false,
+
+    // In this example, assume the billing and rendering providers are the same
+    billing_provider: {
+      first_name: provider.name?.[0]?.given?.[0],
+      last_name: provider.name?.[0]?.family,
+      address: convertAddress(provider.address?.[0]),
+      tax_id: getIdentifier(provider, 'http://hl7.org/fhir/sid/us-ssn'),
+      npi: getIdentifier(provider, 'http://hl7.org/fhir/sid/us-npi'),
+    },
+    rendering_provider: {
+      first_name: provider.name?.[0]?.given?.[0],
+      last_name: provider.name?.[0]?.family,
+      address: convertAddress(provider.address?.[0]),
+      npi: getIdentifier(provider, 'http://hl7.org/fhir/sid/us-npi'),
+    },
+
+    // Copy the information about the service facility
+    service_facility: {
+      organization_name: serviceFacility.name,
+      address: convertAddress(serviceFacility.address?.[0]),
+    },
+    pay_to_address: convertAddress(serviceFacility.address?.[0]),
+    patient: convertPatient(patient),
+    subscriber_primary: {
+      ...subscriber,
+      // '18' - 'Self' (see Candid Health API docs)
+      patient_relationship_to_subscriber_code: '18',
+      insurance_card: convertInsuranceCard(coverage),
+    },
+    diagnoses: convertDiagnoses(encounter),
+
+    // '10' - '' (see Candid Health API docs)
+    place_of_service_code: '10',
+    service_lines: [
+      {
+        procedure_code: encounter.type?.[0] && getCodeBySystem(encounter.type?.[0], 'http://www.ama-assn.org/go/cpt'),
+        quantity: '1',
+        units: 'MJ',
+        charge_amount_cents: 10000,
+        diagnosis_pointers: [0],
+      },
+    ],
+    synchronicity: 'Synchronous',
+  };
+
+  const result = await submitCandidEncounter(
+    candidCodedEncounter,
+    event.secrets['CANDID_API_KEY'].valueString as string,
+    event.secrets['CANDID_API_SECRET'].valueString as string
+  );
+
+  console.log('Received Response from Candid:\n', JSON.stringify(result, null, 2));
+  return result;
+}
+
+/**
+ * Authenticates into the Candid Health API using API key and API secret, and posts the CodedEncounter object to
+ * Candid's /v1/coded_encounters endpoint
+ * @param candidCodedEncounter A JS representation of the CodedEncounter object
+ * @param apiKey  Candid Health API Key
+ * @param apiSecret Candid Health API Secret
+ * @returns
+ */
+async function submitCandidEncounter(candidCodedEncounter: any, apiKey: string, apiSecret: string): Promise<any> {
+  // Get a Bearer Token
+  const authResponse = await fetch('https://api-staging.joincandidhealth.com/api/v1/auth/token', {
```

Comment:
maybe move this to a constant like CANDID_BASE_URL or something like that and pass it into the function?

e.g. 
`const authResponse = await fetch(CANDID_BASE_URL + '/api/v1/auth/token'...`